packages/integration-tests/api/distribution.js

```diff
@@ -231,6 +231,7 @@ async function getTEARequestJwtToken(
     exp: expirationTime,
   }, jwtPrivateKey, {
     algorithm: JwtAlgo,
+    allowInsecureKeySizes: true,
```

Comment:
What's this?

Reply:
It's part of the jsonwebtoken v8 to v9 migration. The length of the key we are using is less than the new required length.
https://github.com/nasa/cumulus/pull/3203/files#r1055865223